cinema_service/settings.py

```diff
@@ -45,6 +44,7 @@
     "rest_framework",
     "rest_framework.authtoken",
     "debug_toolbar",
+    "drf_spectacular",
```

Comment:
The `drf_spectacular` app is included in `INSTALLED_APPS`, which is good for API documentation. Ensure that the documentation is kept up-to-date with the API changes.